bin/setup

```diff
@@ -30,7 +30,4 @@ chdir APP_ROOT do
 
   puts "\n== Removing old logs and tempfiles =="
   system! 'bin/rails log:clear tmp:clear'
-
-  puts "\n== Restarting application server =="
-  system! 'bin/rails restart'
```

Comment:
Removed, as we don't have a server. Could of course restart our workers, but I've always found this bit of dubious merit anyway.